openshift/templates/jobs/backups/backup-cron.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: backup-mongodb
+  labels:
+    template: "backup-mongodb-cronjob"
+    cronjob: "backup-mongodb"
+spec:
+  schedule: "0 9 * * *" # 9am UTC = 1am PST
+  concurrencyPolicy: "Forbid"
+  successfulJobsHistoryLimit: 5
+  failedJobsHistoryLimit: 2
+  jobTemplate:
+    metadata:
+      labels:
+        template: "backup-mongodb-job"
+        cronjob: "backup-mongodb"
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          labels:
+            template: "backup-mongodb-job"
+            cronjob: "backup-mongodb"
+        spec:
+          containers:
+            - name: "backup-mongodb-cronjob"
+              image: >-
+                artifacts.developer.gov.bc.ca/bcgov-docker-local/backup-container-mongo:latest
+              command:
+                - "/bin/bash"
+                - "-c"
+                - "/backup.sh -1"
+              volumeMounts:
+              - name: backup-lup-mongodb
+                mountPath: /backups/
+              - name: backup-lup-mongodb-verification
+                mountPath: /var/lib/mongodb/data
+              - name: backup-mongodb-config-volume
+                mountPath: /backup.conf
+                subPath: backup.conf
+              env:
+              - name: BACKUP_STRATEGY
```

Comment:
So I noticed that there are other settings, which have defaults, if not set, but wondering if for transparency it would be good to add them, with the defaults.

Reply:
Yeah, thats part of the reason why I referenced the documentation in the readme. I definitely did this tailored to LUP, but there are a lot of other fields that other places may need.

Main concern with keeping additional unused ENV vars would be the confusion when someone jumps into OpenShift making changes to or troubleshooting decoy items.

Reply:
Yes, if they need to be updated. I would prefer that over deleting and re-deploying the templates in OpenShift as that would also remove the history, etc.

Reply:
if you template-ize this, the `oc process` command handles the delete-redeploy for you. then your file/repo here can be the SSoT and history.